src/test/java/JsonUtilityTest.java

```diff
@@ -113,11 +113,11 @@ public void sendMapToResponse() throws IOException {
   @Test
   public void sendNullToResponse() throws IOException {
     // Sends null to response as json.
-    // Response should contain only a newline character as a result of println.
+    // Response should contain only null and a newline character as a result of println.
     JsonUtility.sendJson(response, null);
     printWriter.flush();
 
-    Assert.assertEquals("\n", stringWriter.toString());
+    Assert.assertEquals("null\n", stringWriter.toString());
```

Comment:
Updated gson to the latest version, null is actually processed and returned in sendJson now.